### ISSUE TITLE
Can we use direct price reductions?

### ISSUE THREAD

**Issue by Luke1982** [tsolucio/PriceCalculation#3]
I see the discount field results in a percentage. Can we also set it so that it applies an absolute discount value?

**Comment by joebordes** [comment]
no. for the moment only percentage is supported and adding another type would break backward compatibility

**Comment by Luke1982** [comment]
As per our discussion on gitter: I'll do a PR and we can assess how to alter it without breaking existing installations
